resources/files/runtime/runtime.sh

```diff
@@ -16,7 +16,7 @@ else
     LIBDIR="$1"
   else
     echo "$1 does not exist" >2
-    exit 1
+    exit 0
```

Comment:
I think that we bypassed this by not running this sh on certain platforms. I don't think this change should affect any of existing platforms